docs/source/guide/persistent_storage.md

```diff
@@ -298,6 +330,27 @@ Set up Google Cloud Storage (GCS) as the persistent storage for Label Studio Ent
       - Value: `projects/_/buckets/heartex-example-bucket-123456`
     - Or, **use a Common Expression Language** (CEL) to specify an IAM condition. For example, set the following: `resource.name.startsWith('projects/_/buckets/heartex-example-bucket-123456')`. See [CEL for Conditions in Overview of IAM Conditions](https://cloud.google.com/iam/docs/conditions-overview#cel) in the Google Cloud Storage guide. 
 
+### Optional: Configure CORS for the GCS bucket
+
+> In the case if you're going to use direct file upload feature and store media files like audio, video, csv you should complete this step.
+
+Set up cross-origin resource sharing (CORS) access to your bucket. See [Configuring cross-origin resource sharing (CORS)](https://cloud.google.com/storage/docs/configuring-cors#configure-cors-bucket) in the Google Cloud User Guide. Use or modify the following example:
```

Comment:
You don't have to call out cross-origin resource sharing (CORS) but you can use CORS here. The reason because you abbreviate it once at the beginning of the file and use the short terms throughout the file.